src/main/java/com/pollapp/entity/Poll.java

```diff
@@ -0,0 +1,75 @@
+package com.pollapp.entity;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+@Entity
+@Table(name = "poll")
+public class Poll {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String question;
+
+    @JoinColumn(name = "account_id")
+    @ManyToOne(fetch = FetchType.EAGER)
+    private UserAccount userAccount;
+
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "poll", cascade = {CascadeType.MERGE, CascadeType.REMOVE})
+    private List<Answer> answers;
```

Comment:
I think it may be easier in future, not to use plural form as a name. (e.g. List<Answer> answer)

Reply:
well i prefer to use plural when type is a collection, but i think we have to discuss it in a group whether we use these name or those